src/main/java/core/basesyntax/model/objects/Plant.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model.objects;
+
+import java.util.Objects;
+
+public class Plant {
+    protected String name;
+    protected Integer balance;
```

Comment:
You cannot store information about the quantity of plants inside plant by itself